src/feature/dirauth/process_descs.h

```diff
@@ -36,4 +36,6 @@ void dirserv_set_node_flags_from_authoritative_status(node_t *node,
 
 int dirserv_would_reject_router(const routerstatus_t *rs);
 
+int dirserv_router_has_valid_address(routerinfo_t *ri);
```

Comment:
We use a TOR_UNIT_TESTS guard macro for functions that are static, but used in the unit tests:
```suggestion
#ifdef TOR_UNIT_TESTS
STATIC int dirserv_router_has_valid_address(routerinfo_t *ri);
#endif /* defined(TOR_UNIT_TESTS) */
```